tcf_website/management/commands/load_semester.py

```diff
@@ -252,29 +252,49 @@ def load_section(
             units,
             section_type):
 
+        unique_params = {}
         params = {}
-        fields = {
+
+        unique_fields = {
             'sis_section_number',
             'semester',
+        }
+        fields = {
             'course',
             'topic',
             'units',
-            'section_type'}
+            'section_type'
+        }
+
         for k, v in locals().items():
+            if k in unique_fields and not pd.isnull(v):
+                unique_params[k] = v
             if k in fields and not pd.isnull(v):
                 params[k] = v
 
         # print(locals())
-        section, created = Section.objects.get_or_create(
-            **params
-        )
+        # section, created = Section.objects.update_or_create(
+        #     sis_section_number=unique_params['sis_section_number'],
```

Comment:
Why not `**unique_params`?

Reply:
That's used later on; this section is a bad attempt that I commented out and meant to remove.